### ISSUE TITLE
Spectre invocation bug

### ISSUE THREAD

**Issue by dan-fritchman** [Vlsir/Vlsir#76]
Current `main` (and v4.0.0) `vlsirtools.spice.spectre` invocations are all broken. 

```
>           raise SimError(sim=self, stdout=stdout, stderr=stderr)
E           vlsirtools.spice.spice.SimError:
E           rundir: /bwrcq/B/dan_fritchman/dev/VlsirWorkspace/Vlsir/VlsirTools/scratch
E           stdout: b"\nSpectre (R) Circuit Simulator\nVersion 19.1.0.064 64bit -- 6 Sep 2019\nCopyright (C) 1989-2019 Cadence Design Systems, Inc. All rights reserved\n        worldwide. Cadence and Spectre are registered trademarks of Cadence\n        Design Systems, Inc. All others are the property of their respective\n        holders.\n\nIncludes RSA BSAFE(R) Cryptographic or Security Protocol Software from RSA\n        Security, Inc.\n\nUser: dan_fritchman   Host: bwrcr720-3.EECS.Berkeley.EDU   HostID: 2080163F  \n        PID: 4163\nMemory  available: 665.8007 MB  physical: 202.5644 GB\nLinux   : Red Hat Enterprise Linux Server release 7.9 (Maipo)\nCPU Type: Intel(R) Xeon(R) CPU E5-2690 0 @ 2.90GHz\nAll processors running at 3300.0 MHz\n        Socket: Processors\n        0:       0,  2,  4,  6,  8, 10, 12, 14\n        1:       1,  3,  5,  7,  9, 11, 13, 15\n        \nSystem load averages (1min, 5min, 15min) : 57.8 %, 57.6 %, 58.6 %\n\n\nError found by spectre.\n    ERROR (SPECTRE-132): The input file '' has been re-specified as\n        'netlist.scs'.\n        Type `spectre -help' in a terminal window for more information.\n"
E           stderr: b''

vlsirtools/spice/base.py:115: SimError
```

#75 has the fix, which is a one-liner, using `shlex.split` instead of a built-in string-split. 

Real problem is, this doesn't get tested.  
The only way I think we could test this is with the (somewhat new) self-hosted GH Actions runners, on research infrastructure: 
https://docs.github.com/en/actions/hosting-your-own-runners/managing-self-hosted-runners/about-self-hosted-runners

**Comment by dan-fritchman** [comment]
Probably gotta do a v4 patch release for this.

**Comment by dan-fritchman** [comment]
Merged #75 into main. 
It is a one-line change. 
Until we cut a release v5, going to leave this open, since we may want/ need to patch it onto v4.
